src/Nexus.ts

```diff
@@ -155,6 +178,10 @@ class Nexus {
   private mQuota: Quota;
   private mValidationResult: types.IValidateKeyResponse;
   private mRateLimit: { daily: number, hourly: number } = { daily: 1000, hourly: 100 };
+  private oAuthCredentials: types.IOAuthCredentials;
```

Comment:
We prefix private member variables with an "m" followed by an upper-case character, so it should be "mOAuthCredentials" and "mOAuthConfig".